apps/vessel-history/features/search/search.thunk.ts

```diff
@@ -59,10 +60,10 @@ export const fetchData = async (
     limit: RESULTS_PER_PAGE,
     offset,
     query: serializedQuery,
-    useTMT: true,
+    'use-tmt': true,
```

Comment:
Have we changed this param in the api?

Reply:
every camelCase param is now kebab-case in the API
@rdgfuentes do you think is there any other one I could miss?